### PR TITLE
Convert src/plugins/plugin_builder.py using 2to3

### PR DESCRIPTION
**Pull request by lbartoletti**
## Description
Missing in previous PR: src/plugins/plugin_builder.py is now a really python3 script

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [X] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [X] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: src/plugins/plugin_builder.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def plugin_file(pluginDir, file):
 # will make will wreak havoc....
 
 myDir = os.getcwd()
-print("Checking that we are in the <qgis dir>/src/plugins/ directory....", end='')
+print("Checking that we are in the <qgis dir>/src/plugins/ directory....", end=' ')
```
Comment (m-kuhn) [inline src/plugins/plugin_builder.py]: What's the reason for the space in `end=' '`?

Comment (lbartoletti) [reply src/plugins/plugin_builder.py]: My bad

**Comment by m-kuhn** [comment]
Interesting, what is this script used for? It's in the C++ source code. And I think plugin builder is distributed through the plugin system, not within core. Are you using this script for something?

**Comment by luipir** [comment]
Uao! plugin builder is in the qgis code? I didn't realised! I suppose it has to be removed!

**Comment by lbartoletti** [comment]
I don't use it. I have see it when searching plugin builder ("true" python plugin)
I thought it was for C++ plugin.

**Comment by lbartoletti** [comment]
Close, it will be removed with #6768

**Pull request closed without merging**